### DISCUSSION TITLE
Bot Commands

### DISCUSSION THREAD

**Discussion by LontzroFF** [Joe91/fun-bots#206] [Q&A] [answered]
First of all, a HUGE thank you for this mod! I am having such a blast, so many throwbacks...  anyways. Since version 2.4.0 I have bumped against the bot command commo rose. I say bumped against because I use L-Alt for crouching in all FP games I play. Is there a way to assign it to another key?

Thanks for your help! And again, thank you so much for creating this!

**Comment by Joe91** [comment, marked as answer]
Hi,

Yeah, it is possible to change that key. All thats needed is to modify these two lines:

https://github.com/Joe91/fun-bots/blob/master/ext/Client/UIClient.lua#L298

https://github.com/Joe91/fun-bots/blob/master/ext/Client/UIClient.lua#L300

I will maybe make this possible with the config or the registry soon...

You can find the valid names here:
https://docs.veniceunleashed.net/vext/ref/fb/inputdevicekeys/

Have fun
Joe

**Reply by LontzroFF** [reply]
Thank you very much @Joe91!
